Biahun_Uladzimir/02.Git.local/02.Git.local.md

```diff
@@ -0,0 +1,74 @@
+ 1003  git init
+ 1004  git config --global user.name "Biahun Uladzimir"
+ 1005  git config --global user.email "begun74@gmail.com"
+ 1006  git config user.name "Biahun Uladzimir"
+ 1007  date >> master.txt
+ 1008  git add master.txt
+ 1009  git commit -a -m "Init master"
+ 1010   date >> master.txt
+ 1011  git commit -a -m " === 2 commit ==="
+ 1012  git log --oneline --decorate --graph --all
+ 1013  git checkout -b dev
+ 1014  git checkout master
+ 1015  date >> master.txt
+ 1016  git commit -a -m " === 3 commit ==="
+ 1017  date >> master.txt
+ 1018  git commit -a -m " === 4 commit ==="
+ 1019  git checkout dev
+ 1020  date >> dev.txt
+ 1021  git add dev.txt
+ 1022  git commit -a -m " === 5 commit ==="
+ 1023  date >> dev.txt
+ 1024  git commit -a -m " === 6 commit ==="
+ 1025  git status
+ 1026  git branch
+ 1027  git checkout -b features/do_one
+ 1028  git checkout master
+ 1029  date >> master.txt.txt
+ 1030  git commit -a -m " === 7 commit ==="
+ 1031  date >> master.txt
+ 1032  git commit -a -m " === 7 commit ==="
+ 1033  rm master.txt.txt
+ 1034   date >> master.txt
+ 1035  git commit -a -m " === 8 commit ==="
+ 1036  git checkout dev
+ 1037  date >> dev.txt
+ 1038  git commit -a -m " === 9 commit ==="
+ 1039  date >> dev.txt
+ 1040  git commit -a -m " === 10 commit ==="
+ 1041  git checkout features/do_one
+ 1042  date >> features_do_one.txt
+ 1043  git checkout master
+ 1044  git checkout -b hotfix/we_gonna_die
+ 1045   git checkout master
+ 1046   date >> master.txt
+ 1047  git commit -a -m " === 11 commit ==="
+ 1048   date >> master.txt
+ 1049   git commit -a -m " === 12 commit ==="
+ 1050  git checkout  hotfix/we_gonna_die
+ 1051  date >> hotfix_we_gonna_die.txt
+ 1052  git commit -a -m " === 13 commit ==="
+ 1053  git add hotfix_we_gonna_die.txt
+ 1054  git commit -a -m " === 13 commit ==="
+ 1055  git checkout master
+ 1056  git merge hotfix/we_gonna_die
+ 1057  ls -la
+ 1058  cat master.txt
+ 1059  cat hotfix_we_gonna_die.txt
+ 1060  git branch -r hotfix/we_gonna_die
+ 1061  git branch -d hotfix/we_gonna_die
+ 1062  git checkout dev
+ 1063  ls -la
+ 1064  git branch -v
+ 1065  git merge features/do_one
+ 1066  ls -la
+ 1067* git branch -d dev
+ 1068  git branch -v
+ 1069  git checkout master
+ 1070  git merge master
+ 1071  git checkout master
+ 1072  ls -la
+ 1073  git merge dev
+ 1074  ls -la
+ 1075  history
+1111  git log --oneline --decorate --graph --all
```

Comment:
I need output on this command, even short `git log --oneline`
Your branches should have:
master (init) 6 commits
dev (from master)  5 commits
features/do_one (from dev) 6 commits
hotfix/we_gonna_die (from master) 3 commits